projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-social-media-connections/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+
+/**
+ * Hooks to deal with the social media connections.
+ *
+ * @returns {Function} Social media connection handler.
+ */
+export default function useSocialMediaConnections() {
+	const { editPost } = useDispatch( editorStore );
+
+	const connections = useSelect(
+		select => select( editorStore ).getEditedPostAttribute( 'jetpack_publicize_connections' ),
+		[]
```

Comment:
Does an empty array mean that it would only be selected once? If so, would that break the connection refresh PR? The toggle isn't working for me, and this could be why.

Reply:
👀 looking...

Reply:
The connection PR adds a new `getConnections()` selector that is used instead of this one.

Reply:
> The toggle isn't working for me, and this could be why.

It was an error that I introduced when refactoring. Fixed here https://github.com/Automattic/jetpack/commit/4582e5d59914d40a15fa1f7845ab011caa3a2c38